src/components/QrScan/QrScan.js

```diff
@@ -106,12 +174,8 @@ const QrScan = () => {
         </>
       )}
       <Grid item xs={4}>
-        <QrReader
-          className={classes.qrScanner}
-          onError={handleError}
-          onScan={handleScan}
-          showViewFinder={false}
-        />
+        {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+        <video id="test" className={classes.qrScanner} ref={videoRef} />
```

Comment:
You shouldn't need to disable eslint here if you add the `muted` attribute to the video component.

See https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/c460a8b896adf42b689b35b6820d634e552f290b/docs/rules/media-has-caption.md#succeed

```suggestion
        <video
          muted
          id="test"
          className={classes.qrScanner}
          ref={videoRef}
        />
```